### DISCUSSION TITLE
Pas de bouton pour changer de camera (iphone) - Résolu

### DISCUSSION THREAD

**Discussion by PhilMncn** [seb-cogez/QCMCam#7] [General]
Bonjour,

Merci pour l'appli.

Je n'ai pas le bouton permettant de changer de caméra sur iphone (dernière version ios). J'ai essayé firefox, safari et chrome.
Le bouton n'apparaît pas même en tournant l'écran.

Merci
Philippe

**Comment by PhilMncn** [comment]
J'ai vu la réponse à la question dans un autre forum que j'aurais dû lire avant de poster !
Je mets le lien vers la vidéo qui résout le problème.
https://tube-numerique-educatif.apps.education.fr/w/p/1iP6p5WAzH9TWZF5DPX3YG
Merci

**Comment by seb-cogez** [comment]
Bonjour,

Ce tuto vidéo est disponible depuis la page d'accueil de QCMCam, lien peerTube de ma chaine, j'ajoute de suite une petite note dans la documentation. Merci pour ce retour

Bien cordialement
